### ISSUE TITLE
Any free webhosts that work with this panel?

### ISSUE THREAD

**Issue by zahin45** [UnamSanctam/UnamWebPanel#88]
It worked with 000webhost,  everything worked fine, the miners showed etc, but they took my site down cuz yeah
Ive tried with other sites, and the panels show but the miners dont,

**Comment by UnamSanctam** [comment]
Well no free webhost will really allow too much traffic for obvious reasons, but any webhost that supports PHP also supports this web panel. Just make sure that the database (unamwebpanel.db) has write permissions (and maybe the parent folder in some cases) and that the PHP version is 7.4 or above.

**Comment by zahin45** [comment]
How can I give perms in cpanel?

**Comment by UnamSanctam** [comment]
Go to the file manager and modify the permissions there, for it to work they should be something like 644.

**Comment by zahin45** [comment]
![image](https://user-images.githubusercontent.com/67522617/155904081-3f891add-568a-45d0-a581-b3ff53336da8.png)

**Comment by zahin45** [comment]
![image](https://user-images.githubusercontent.com/67522617/155904244-058a213d-21bb-4584-bbf8-890414785a8f.png)

just this file?

**Comment by UnamSanctam** [comment]
Yes you can try that just to see if it's the permissions that's doing it, you can also test if writing to the database works by going to the "Configurations" tab and modifying/creating a new configuration and then refreshing to see if it's saved.

You can also host the web panel on a VPS or behind a VPN if you have that as well.

**Comment by zahin45** [comment]
what configurations tab?

**Comment by UnamSanctam** [comment]
On the web panel.

**Comment by zahin45** [comment]
yes it saves

**Comment by UnamSanctam** [comment]
Are you using http or https? And if you're using https does the domain have a certificate? Most free webhosts do for the subdomains but I have to ask.

**Comment by zahin45** [comment]
when I go to the website, the url is http
